opensrp-app/src/test/java/org/smartregister/EnvironmentManagerTest.java

```diff
@@ -0,0 +1,82 @@
+package org.smartregister;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Answers;
+import org.mockito.Mock;
+import org.robolectric.util.ReflectionHelpers;
+import org.smartregister.domain.Environment;
+import org.smartregister.util.JsonFormUtils;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+
+/**
+ * Created by Ephraim Kigamba - nek.eam@gmail.com on 27-04-2021.
+ */
+public class EnvironmentManagerTest extends BaseRobolectricUnitTest {
+
+    @Mock(answer = Answers.CALLS_REAL_METHODS)
+    private EnvironmentManager environmentManager;
+
+    @Test
+    public void getEnvironments() {
+        HashMap<String, Environment> environmentHashMap = new HashMap<>();
+        Environment env1 = new Environment();
+        Environment env2 = new Environment();
+
+        environmentHashMap.put("goldsmith-stage", env1);
+        environmentHashMap.put("reveal-stage", env2);
+
+        ReflectionHelpers.setField(environmentManager, "environments", environmentHashMap);
+
+        // Call the method under test
+        Collection<Environment> environments = environmentManager.getEnvironments();
+
+        Assert.assertEquals(2, environments.size());
+        Assert.assertTrue(environments.contains(env1));
+        Assert.assertTrue(environments.contains(env2));
+
+    }
+
+    @Test
+    public void getEnvironment() {
```

Comment:
The method name could be a little bit more descriptive to understand what's being tested

Reply:
The test just tests the getter for the environment and the method description describes that. We do not have a variation or special case on this method. Do you have a suggestion?

Reply:
Okay. That's fine then